### PR TITLE
Wants the original non-parsed JWT and not an IDToken instance.

### PR DESCRIPTION
**Pull request by rohe**
- [ ] Any changes relevant to users are recorded in the `CHANGELOG.md`.
- [ ] The documentation has been updated, if necessary.
- [ ] New code is annotated.
- [ ] Changes are covered by tests.
---

### CODE REVIEW

**Comment by rohe** [comment]
I think mypy is wrong !

**Comment by tpazderka** [comment]
Yes, it cannot recognize the dynamically created attributes of the Client. You can probably add the ignore statement on the offending line (`type: ignore`).

I will go through the rest of the PR later today.

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ The format is based on the [KeepAChangeLog] project.
 - [#679] Make `state` optional in `EndSessionRequest`
 - [#683] Fix basic_auth with client password
 - [#698] `state` in `EndSessionRequest` request args and kwargs different
+- [#708] Wants the original non-parsed JWT and not an IDToken instance.
```
Comment (tpazderka) [inline CHANGELOG.md]: This needs to go to the unreleased section as it is going to be a bugfix.

File: tests/test_oic.py [hunk 2]
```diff
@@ -182,19 +181,14 @@ def test_access_token_request(self):
         assert _eq(resp.keys(), ["token_type", "state", "access_token", "scope"])
 
     def test_access_token_request_with_custom_response_class(self):
```
Comment (tpazderka) [inline tests/test_oic.py]: Please leave this test unchanged. This is here to ensure that the custom response class is honored and is not actually related to the content of the response.

Comment (rohe) [reply tests/test_oic.py]: Done

File: CHANGELOG.md (hunk 1 above)
Comment (rohe) [reply CHANGELOG.md]: Done

File: tests/test_oic.py [hunk 3]
```diff
@@ -224,7 +223,6 @@ class CustomMessageFactory(OIDCMessageFactory):
                     "access_token": "token",
                     "scope": "openid",
                     "state": "state0",
-                    "raw_id_token": "token",
```
Comment (tpazderka) [inline tests/test_oic.py]: This line should also remain here.

Comment (rohe) [reply tests/test_oic.py]: Damn, right you are !

**Pull request merged**